cranelift/assembler-x64/meta/src/instructions/addpd.rs

```diff
@@ -0,0 +1,15 @@
+use crate::dsl::{
+    align, fmt, inst, r, rex, rw, vex, w, Feature::*, Inst, Location::*, VexLength::*, VexMMMMM::*, VexPP::*,
+};
+
+pub fn list() -> Vec<Inst> {
+    vec![
+        inst("addpd", fmt("A", [rw(xmm1), r(align(rm128))]), rex([0x66, 0x0F, 0x58]).r(), _64b | compat | sse),
```

Comment:
This duplicates one of the definitions in `add.rs`, right? Let's just keep all the addition in that file to avoid this kind of mistake.

Reply:
For the final vex support I removed the one in add.rs (but put it back for this pr). The reason is simple, I was organizing the instructions the same way they are organized in the manual by opcode (specifically Vol 2A 3-34). The ADDPD opcode instruction section covers rex, vex, and evex. What do you think about this organization of grouping instructions consistent with the manual?